### PR TITLE
vp-17-changed the Hover colour in App.css file

### PR DESCRIPTION
**Pull request by chris-reejoy**

### CODE REVIEW

File: calculator-app/src/App.css [hunk 1]
```diff
@@ -54,9 +54,9 @@ ul.ul-numbers {
 }
 
 .button-cldel:hover {
-  background-color: aqua;
+  background-color: orange;
 }
 
 .button-numbers:hover {
-  background-color: aqua;
+  background-color: orange;
```
Comment (prabhugberlin) [inline calculator-app/src/App.css]: Use color code

File: calculator-app/src/ButtonClear.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <div className="App">
+      <header className="App-header">
+        <div>
+      <button type="button" class="Buttonclear button">Clear</button > 
+      <button type="button" class="Buttonclear button">Del</button > 
+      </div>
+        {/*
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
```
Comment (prabhugberlin) [inline calculator-app/src/ButtonClear.js]: remove unwanted commented lines

File: calculator-app/src/ButtonClear.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import './App.css';
+
+function ButtonClear() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <div>
```
Comment (prabhugberlin) [inline calculator-app/src/ButtonClear.js:7]: Try to avoid <div>and <header> tag

File: calculator-app/src/App.css (hunk 1 above)
Comment (chris-reejoy) [reply calculator-app/src/App.css]: changed with color code

File: calculator-app/src/ButtonClear.js (hunk 2 above)
Comment (chris-reejoy) [reply calculator-app/src/ButtonClear.js]: done

**Review by prabhugberlin — APPROVED** [review]

**Review by RanjiniEswari — APPROVED** [review]

**Review by Mutheeswari — APPROVED** [review]

**Pull request merged**